src/pages/besquare/components/sections/_what-is_besquare.js

```diff
@@ -37,11 +37,11 @@ const WhatIsBeSquare = () => {
     return (
         <Section>
             <ContentContainer>
-                <ContentWrapper>
-                    <Title as="h2" text_align={'left'} max_width={['282px']} margin={['0', '0 auto']}>
+                <ContentWrapper margin={'120px'}>
+                    <Title as="h2" text_align={'left'} max_width={['450px']} margin={['0', '0 auto']}>
                         {what_is_be_square.title}
                     </Title>
-                    <TextWrapper max_width={['486px', '292px']} margin={['8px 0 0', '16px 0']}>
+                    <TextWrapper max_width={['486px', '328px']} margin={['8px 0 0', '16px 0 16px 0']} >
```

Comment:
```suggestion
                    <TextWrapper max_width={['486px', '328px']} margin={['8px 0 0', '16px 0']} >
```

Reply:
please use two rule sets if the padding/margin value is repeated. For example, for this situation `16px 0 16px 0` can also be written as `16px 0`

Reply:
ok